src/cpp/RiderLink/Source/RD/src/rd_framework_cpp/src/main/util/hashing.h

```diff
@@ -30,7 +30,7 @@ constexpr hash_t getPlatformIndependentHash(char const (&that)[N], constexpr_has
 
 constexpr hash_t getPlatformIndependentHash(string_view that, constexpr_hash_t initial = DEFAULT_HASH)
 {
-	return static_cast<hash_t>(hashImpl(initial, &that[0], &that[that.length()]));
+	return static_cast<hash_t>(hashImpl(initial, &that[0], &that[that.length() - 1] + 1));
```

Comment:
Can you explain what's happening here so I don't need to open the source code? :) We're taking an address "next after the last one"?

Reply:
&that[that.length()] - is the end pointer, i.e. invalid pointer after string.
In debug string_view::operator[] has debug checks if index is out of range and when evaluating in constexpr context such access to end pointer cause compilation error.
So I changed retrieval of end pointer as pointer to last element + 1